lib/gtfs/trip.ex

```diff
@@ -20,11 +21,14 @@ defmodule Gtfs.Trip do
     :route_id
   ]
 
-  @spec from_csv_row(%{required(String.t()) => String.t()}) :: t()
+  @spec from_csv_row(Csv.row()) :: t()
   def from_csv_row(row) do
     %__MODULE__{
       id: row["trip_id"],
       route_id: row["route_id"]
     }
   end
+
+  @spec in_route_id_set?(Csv.row(), MapSet.t(id())) :: boolean
```

Comment:
Should have type `MapSet.t(Route.Id())`
```suggestion
  @spec in_route_id_set?(Csv.row(), MapSet.t(Route.id())) :: boolean
```